src/api/public/apidocs-new/paths/source_project_name_cmd_createpatchinfo.yaml

```
@@ -0,0 +1,80 @@
+post:
+  summary: Creates a patchinfo inside the given project
+  description: |
+    Creates a new patchinfo package, in the given project, containing a '_patchinfo' file.
+    The file will include all the issues in other project's sources, if any.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: query
+      name: name
+      schema:
+        type: string
+      description: Name of the patchinfo package. If not passed, 'patchinfo' is the default value.
+      example: BaseDistro_patchinfo
+    - in: query
+      name: comment
+      schema:
+        type: string
+      description: Comment to be included in the 'summary' XML tag of the patchinfo.
+      example: Security update for the Linux Kernel
+    - in: query
+      name: force
+      schema:
+        type: string
+      description: |
+        If the patchinfo with the given name already existed and the force parametter is passed,
```

```suggestion
        If the patchinfo with the given name already existed and the force parameter is passed,
```
Typo in `parameter`.

---

Corrected!